src/router/index.js

```diff
@@ -24,12 +26,28 @@ const routes = [
   {
     path: "/",
     name: "Home",
-    component: Home
-  },
+    get component() {
+      const x=1; // emulating the value of the condition check for the sign in.
```

Comment:
Please remove component switching logic based on authentication from routes to the view.